apps/opencs/view/doc/view.hpp

```diff
@@ -66,6 +66,9 @@ namespace CSVDoc
 
             void closeEvent (QCloseEvent *event);
 
+            QAction* createMenuEntry(CSMWorld::UniversalId::Type type, QMenu* menu, const char* shortcutName);
```

Comment:
Good idea to factor out the menu item creation into a function. I should have done that in the first place.